plugins/attention-and-distributed-packing/src/fms_acceleration_aadp/framework_plugin_multipack.py

```diff
@@ -44,8 +44,7 @@ def __init__(
         )
 
         # multipack settings
-        self._effective_batch_size = multipack["effective_batch_size"]
-        self._max_batch_len = multipack["max_number_tokens"]
+        self.num_workers = multipack["num_workers"]
```

Comment:
```suggestion
        self.num_processes = self._check_config_and_maybe_check_values(
            key="training.dataloader.multipack.num_processes",
        )
```

then you can delete off `multipack` above

Reply:
also delete `self._collate_fn` below